src/Tizen.NUI/src/internal/WebView/WebContext.cs

```diff
@@ -155,25 +191,151 @@ public string CertificateFilePath
             }
             set
             {
-                certificateFilePath = value;
                 Interop.WebContext.SetCertificateFilePath(SwigCPtr, value);
+                if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
+                certificateFilePath = value;
             }
         }
 
         /// <summary>
         /// Disable cache or not.
         /// </summary>
         [EditorBrowsable(EditorBrowsableState.Never)]
-        public bool DisableCache
+        public bool CacheEnabled
         {
             get
             {
-                return disableCache;
+                return Interop.WebContext.IsCacheEnabled(SwigCPtr);
             }
             set
             {
-                disableCache = value;
-                Interop.WebContext.DisableCache(SwigCPtr, value);
+                Interop.WebContext.EnableCache(SwigCPtr, value);
+                if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
+            }
+        }
+
+        /// <summary>
+        /// Get context certificate file.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public string ContextCertificateFile
```

Comment:
Does it return a file name? (With full path?) Or does it return the content of certificate file in string?